app/jobs/reports/monthly_key_metrics_report.rb

```diff
@@ -53,7 +54,7 @@ def reports
         active_users_count_report.active_users_count_emailable_report,
         # Total Annual Users - LG-11150
         total_user_count_report.total_user_count_emailable_report,
-        # Proofing rate(s) (tbd on this one pager) - LG-11152
+        proofing_rate_report.proofing_rate_emailable_report,
```

Comment:
did we confirm we're going to remove the other proofing report? and if so, is that going to be a separate PR?

Reply:
No, there were too many questions - so I'm separating that out.  We'll have two of the three proofing reports in the MKMR after this is merged and then we can ask Cale about how he wants to combine info from the three proofing reports.